lms/static/scripts/frontend_apps/components/SubmitGradeForm.js

```diff
@@ -20,18 +20,46 @@ import ValidationMessage from './ValidationMessage';
 export default function SubmitGradeForm({ disabled = false, student }) {
   const [showError, setShowError] = useState(false); // Is there a validation error message to show?
   const [gradeErrorMessage, setGradeErrorMessage] = useState(''); // The actual validation error message.
+  const [grade, setGrade] = useState('');
 
-  const [networkError, setNetworkError] = useState(''); // if there is an error when submitting grade
-  const [requestStatus, setRequestStatus] = useState(''); // ajax request state, one of ('', 'fetching', 'error', 'done')
+  const [networkError, setNetworkError] = useState(''); // If there is an error when submitting grade
+  const [requestStatus, setRequestStatus] = useState(''); // Ajax request state, one of ('', 'fetching', 'error', 'done')
+  const [fetchStatus, setFetchStatus] = useState(''); // Ajax request state, one of ('', 'fetching', 'error', 'done')
 
   const { authToken } = useContext(Config);
 
+  // Used to handle keyboard input effects.
+  const refInput = useRef(null);
+
   useEffect(() => {
     setRequestStatus(''); // clear out network status
```

Comment:
Similar to the comment about `requestStatus` being confusing...this feels vague because it's not clear what network status is being cleared out.